backend/src/test/java/eu/bbmri_eric/negotiator/integration/api/v3/NotificationControllerTest.java

```diff
@@ -62,4 +62,68 @@ void getNotifications_3new_Ok() throws Exception {
         .andExpect(content().contentType(MediaType.APPLICATION_JSON))
         .andExpect(jsonPath("$.length()", is(3)));
   }
+
+  @Test
+  @WithUserDetails("admin")
+  void getNotificationTemplate_validTemplateName_Ok() throws Exception {
+    mockMvc
+        .perform(MockMvcRequestBuilders.get("/v3/notifications/template/footer"))
+        .andExpect(status().isOk())
+        .andExpect(content().contentType("application/xhtml+xml;charset=UTF-8"));
+  }
+
+  @Test
```

Comment:
I dont see any test for validating the the templates are actually used in the emails